jwala-services/src/test/java/com/cerner/jwala/service/jvm/impl/JvmServiceImplTest.java

```diff
@@ -1198,6 +1198,13 @@ public void testUpgradeJDK() throws CommandFailureException, IOException {
 		when(Config.mockResourceService.getResourceContent(resourceIdentifier)).thenReturn(resourceContent);
 		Jvm response = jvmService.upgradeJDK(mockJvm.getJvmName(), Config.mockUser);
 		assertEquals(response.getJvmName(), mockJvm.getJvmName());
+
+		/**
```

Comment:
Remove this comment.

Some of the the verification steps you need to assert are actually detailed in the comment:
 - resource service fetches the setenv.bat
 - resource service generates the setenv.bat

Please add those verify statements for the mockResourceService.